website/docs/concepts/event-bridge/integrations.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 5
+---
+
+# Integrations
+
+Functionless supports integrations between some AWS services and Event Bridge. Send events to an `EventBus` using the `putEvents` API and send events to other resources using the `.pipe` method.
+
+| Resource       | From `EventBus` | To `EventBus` |
+| -------------- | --------------- | ------------- |
+| _via_          | `pipe`          | `putEvents`   |
+| Lambda         | &#x2705;        | &#x2705;      |
+| Step Functions | &#x2705;        | &#x2705;      |
+| EventBus       | &#x2705;        | &#x2705;      |
+| App Sync       |                 | Coming Soon   |
+| API Gateway    |                 | Coming Soon   |
+
+See [issues](https://github.com/functionless/functionless/issues?q=is%3Aissue+is%3Aopen+label%3Aevent-bridge) for progress or create a new issue in the form `Event Bridge + [Service]`.
+
+## From `EventBus` using `pipe`
+
+```ts
+new EventBus(stack, "bus")
+  .when("onSignUp", (event) => event.source === "lambda")
+  // send an event to a lambda
+  .pipe(
+    new Function(stack, "func", async (event) => {
+      console.log(event.id);
+    })
+  );
+```
+
+### Escape Hatches
+
+If a target isn't supported by Functionless, `.pipe` supports [any target supported by EventBridge](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_events_targets-readme.html).
+
+```ts
+const logGroup = new aws_logs.LogGroup(this, "MyLogGroup", {
+  logGroupName: "MyLogGroup",
+});
+
+// use the pipe callback escape hatch to pipe to a cloudwatch log group (which functionless doesn't natively support, yet)
+new EventBus()
+  .when((event) => event.source === "lambda")
```

Comment:
Should this have scope and/or id?

Reply:
Yep, though I made scope optional for when because we can use the bus as the scope.

Reply:
Found a few more too.